app/entry.server.tsx

```diff
@@ -1,6 +1,6 @@
 import { PassThrough } from 'node:stream'
+import { contentSecurity } from '@nichtsam/helmet/rules/content/index'
```

Comment:
This is a nitpick, but is there any chance you could add exports config in your package.json so that we don't have to dive deep into your package? Instead, this could be more like this for example:

```suggestion
import { contentSecurity } from '@nichtsam/helmet/content'
```

Reply:
I also didn't like that, but didn't know how to path them better, thanks for the idea!
I will update the package to simplify the import path.